src/pydicom/pixels/utils.py

```diff
@@ -1738,6 +1738,188 @@ def reshape_pixel_array(ds: "Dataset", arr: "np.ndarray") -> "np.ndarray":
     return arr
 
 
+def set_pixel_data(
+    ds: "Dataset",
+    arr: "np.ndarray",
+    photometric_interpretation: str,
+    bits_stored: int,
```

Comment:
Maybe make these keyword-only arguments?

Reply:
I think positional is fine here, they're both required